### ISSUE TITLE
Add Documentation and a Walkthrough

### ISSUE THREAD

**Issue by meyerzinn** [NucleusPowered/QuickStartModuleLoader#5]
I've been looking into using QSML but unfortunately Nucleus is such an advanced plugin it's not helping. It'd be great to see a barebones implementation plugin to demonstrate how to use it. Also, @dualspiral please come on to IRC 😄

**Comment by dualspiral** [comment]
Yeah, I realise that, so I've actually started _some_ docs, but there are a couple of things I need to do. Yeah, Nucleus does some advanced stuff with it, and I'm looking at making QSML a bit more general.

I can't pop onto IRC right now for reasons I can't quite go into, but I'll try to remember to get on it in the near future.

**Comment by parlough** [comment]
I'm curious as what would be needed to make it more general in your mind?
